azure-pipelines.yml

```diff
@@ -102,7 +102,8 @@ parameters:
         aksKeyVault: "dtscftptl"
         terraformSubscriptionID: "1baf5470-1c3e-40d3-a6f7-74bfbce4b348"
       - env: "prod"
-        dependsOn:  "aat"
+        dependsOn:
+         
         serviceConnection: "DCD-CFTAPPS-PROD"
```

Comment:
Did you mean to remove this dependsOn? @NickAzureDevops

Reply:
accidentally  removed it. all good now